apps/ledger-live-desktop/src/renderer/screens/customImage/Step1ChooseImage.tsx

```diff
@@ -107,10 +119,45 @@ const StepChooseImage: React.FC<Props> = props => {
         ) : null
       }
     >
-      {!isShowingNftGallery ? (
+      {isShowingNftGallery ? (
+        <TrackPage
+          category={analyticsPageNames.chooseNftGallery}
+          type="drawer"
+          flow={analyticsFlowName}
+          refreshSource={false}
+        />
+      ) : (
+        <TrackPage
+          category={analyticsPageNames.chooseImage}
+          type="drawer"
+          flow={analyticsFlowName}
+          refreshSource={false}
+        />
+      )}
```

Comment:
```suggestion
       <TrackPage
          category={isShowingNftGallery ? analyticsPageNames.chooseNftGallery : analyticsPageNames.chooseImage}
          type="drawer"
          flow={analyticsFlowName}
          refreshSource={false}
        />    
```
To avoid duplication, But that's just nitpicking 😁